pkgs/tools/misc/z/default.nix

```
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, lib, stdenv }:
+  stdenv.mkDerivation
+    { pname = "z";
+      version = "1622481819";
+      # ^ there isn't a version, so the UNIX time of the commit is used
```

```suggestion
{ fetchFromGitHub, lib, stdenv }:

stdenv.mkDerivation {
  pname = "z";
  version = "unstable-XXXX-XX-XX";
```

please fill in the iso date. We don't use unix time for such things.